src/pages/game.js

```diff
@@ -142,27 +143,29 @@ export default function Game({data, location}) {
 
    return (
       <>
-         <ScoreAside cardsLeft={usersCards.length} turnNumber={turnCount} wins={roundsWon} />
-
-         {isUsersTurn && !roundWinner && (
-            <UserTurn card={usersTurnCard} slamJams={slamJams}></UserTurn>
-         )}
-
-         {!isUsersTurn && !roundWinner && (
-            <ComputersTurn name={computerName} card={computersTurnCard}></ComputersTurn>
-         )}
-
-         {roundWinner && (
-            <Result
-               computerName={computerName}
-               usersTurnCard={usersTurnCard}
-               computersTurnCard={computersTurnCard}
-               winner={roundWinner}
-               selectedAttribute={selectedAttribute}
-               incrementTurnCount={incrementTurnCount}
-               slamJams={slamJams}
-            ></Result>
-         )}
+         <InfoIcon>
```

Comment:
I would  have just replaced the fragment if you have a wrapper, e.g. `<>` becomes `<InfoIcon>`

Reply:
Good spot. Done :-)